### ISSUE TITLE
Top page feed shows localized string although actual content is not localized

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#865]
```
Top page feed shows localized string although actual content is not localized

The best we can offer should be: localized content shows localized feed string

If this is difficult, we should at least make it easier to find localized 
content and not confuse readers by localized feed string.

Options I can imagine would be:
- show only localized feed / localized search results on localized version and 
ask readers to find more contents on English version
- show feeds English only but mark localized contents if possible.
- show feeds English only without localized content mark. It's much better than 
disappointing readers.
```

Original issue reported on code.google.com by `agek...@google.com` on 3 Apr 2012 at 6:51

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 13e4d5619933.
```

Original comment by `mkwst@google.com` on 3 Apr 2012 at 9:54
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision e346d668c7d6.
```

Original comment by `mkwst@google.com` on 3 Apr 2012 at 10:16
